Makefile

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2025 Norbert Melzer <timmelzer@gmail.com>
+#
+# SPDX-License-Identifier: CC0-1.0
```

Comment:
`reuse` can not read the comment from the `Makefile`, please add an external `.license` file.